aea/cli/common.py

```diff
@@ -248,29 +252,6 @@ def format_items(items):
     return list_str
 
 
-def format_skills(items):
-    """Format list of skills to a string for CLI output."""
-    list_str = ""
-    for item in items:
-        list_str += (
-            "{line}\n"
-            "Public ID: {public_id}\n"
-            "Name: {name}\n"
-            "Description: {description}\n"
-            "Protocols: {protocols}\n"
```

Comment:
Note, @panasevychol you removed the dependency info from the search command output for skills now. That's ok but we should open an issue that dependencies should be shown for skills (protocols, contracts) and agents (skills, contracts, protocols, connections).

Reply:
@DavidMinarsch we discussed that before and you told that we won't need protocols to be shown in skills search results. That just wasn't done that time.
IMO we should have an opened issue about output format. Now it looks a bit poor, maybe we should update it, add more info, better formatting. Will create one.